src/panels/GettingStartedPanel/Tasks/FocusTask.tsx

```diff
@@ -0,0 +1,26 @@
+import { Checkbox } from '@mantine/core';
+
+import { useGetStringPropertyValue } from '@/api/hooks';
+import { NavigationAnchorKey } from '@/util/keys';
+
+interface Props {
+  anchor: string;
+}
+
+export function FocusTask({ anchor }: Props) {
+  const [currentAnchor] = useGetStringPropertyValue(NavigationAnchorKey);
+  const taskCompleted =
+    currentAnchor !== undefined && currentAnchor !== '' && currentAnchor === anchor;
+
+  return (
+    <Checkbox
```

Comment:
Aren't the props for all these checkboxes the same... ? Thinking that it should maybe be a separate component?  E.g. `TaskCheckbox.tsx` and could then be put in the Components folder?

Now it's pretty repetitive, and if once wanted to e.g. change the color for the text for all tasks it would be a pain in the ass... :)  So please make a component for it